### PR TITLE
K8SPG-316: Generate OLM bundles

### PR DESCRIPTION
**Pull request by egegunes**
[K8SPG-316](https://jira.percona.com/browse/K8SPG-316) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/3bb59a5ff/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/3bb59a5ff/demand-backup.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/3bb59a5ff/start-from-backup.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/3bb59a5ff/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/402/commits/3bb59a5ff7677f00c8dc2af4198b2a4fb5b9d714
image: `perconalab/percona-postgresql-operator:PR-402-3bb59a5ff`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/0e9f78c41/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/0e9f78c41/demand-backup.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/0e9f78c41/start-from-backup.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/0e9f78c41/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/402/commits/0e9f78c414141d15c01e62e4d8ae8e6ba473cbd8
image: `perconalab/percona-postgresql-operator:PR-402-0e9f78c41`

File: installers/olm/config/examples/postgrescluster.example.yaml [hunk 1]
```diff
@@ -1,23 +1,42 @@
-apiVersion: postgres-operator.crunchydata.com/v1beta1
-kind: PostgresCluster
+apiVersion: pg.percona.com/v2beta1
+kind: PerconaPGCluster
 metadata:
```
Comment (hors) [inline installers/olm/config/examples/postgrescluster.example.yaml:3]: Should we regenerate it to add finalizers?

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/5487a96ff/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/5487a96ff/demand-backup.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/5487a96ff/start-from-backup.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-402/5487a96ff/monitoring.log) |
| We run 4 out of 4|

commit: https://github.com/percona/percona-postgresql-operator/pull/402/commits/5487a96ff32919da6c959b622c134ee110f281a1
image: `perconalab/percona-postgresql-operator:PR-402-5487a96ff`

**Review by hors — APPROVED** [review]

**Pull request merged**